test/e2e-go/e2e_go_olm_test.go

```diff
@@ -63,6 +63,17 @@ var _ = Describe("Integrating Go Projects with OLM", func() {
 				Expect(err).NotTo(HaveOccurred())
 			}
 
+			By("running the operator bundle using `run bundle` command")
+			runBundleCmd := exec.Command(tc.BinaryName, "run", "bundle", bundleImage, "--namespace", tc.Kubectl.Namespace)
```

Comment:
The command is not working see: 

<img width="1281" alt="Screen Shot 2020-09-24 at 07 31 19" src="https://user-images.githubusercontent.com/7708031/94134369-cd695c00-fe59-11ea-9416-0d912cca5c29.png">

Are you able to test command running with success locally? 
c/c @jmrodri

Reply:
Yes, was trying to debug this error with Jesus. `IsRunningOnKind` works fine, but the run bundle logic always reaches to the remote registry to pull and unpack the image. Jesus is working on finding a way out for that. Meanwhile we decided to push the image to a remote repository, and check if it works. But there is some error in the install plan. Trying to figure that out.

Reply:
Hi @varshaprasad96, 

Since we use the IsRunningOnKind in all tests and in more than one place, IMO it should be centralized such as in order to be easy keep it maintained: https://github.com/operator-framework/operator-sdk/pull/3941

Regards the issue, see that in the above pr; 

STEP: loading the bundle image into Kind cluster
running: kind load docker-image e2e-test/controller-manager-bundle:mbnd --name kind
STEP: running the operator bundle using `run bundle` command
running: operator-sdk run bundle e2e-test/controller-manager-bundle:mbnd --namespace e2e-mbnd-system

Which means that the image was loaded in the Kind. So, the problem shows be a bug in the `operator-sdk run bundle`.

Reply:
@camilamacedo86 yes, there is no error with `IsRunningOnKind`. The implementation logic of run bundle expects the image to be in registry. Even if its loaded in the kind cluster, while unpacking it tries to pull it from registry.